maintenance_plan/views/maintenance_equipment_views.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <!-- equipment : views inheritance -->
+    <record id="hr_equipment_view_form" model="ir.ui.view">
+        <field name="name">equipment.form.inherit</field>
+        <field name="inherit_id" ref="maintenance.hr_equipment_view_form" />
+        <field name="model">maintenance.equipment</field>
+        <field name="arch" type="xml">
+            <div name="button_box" position="inside">
+                <button
+                    name="%(maintenance_plan_equipment_action)d"
+                    type="action"
+                    class="oe_stat_button"
+                    icon="fa-list"
+                >
+                    <field
+                        string="Plans"
+                        name="maintenance_plan_count"
+                        widget="statinfo"
+                    />
+                </button>
+            </div>
+            <xpath expr="//group[@name='maintenance']" position="attributes">
+                <attribute name="attrs">{'invisible': 1}</attribute>
+            </xpath>
+            <xpath expr="//group[@name='maintenance']/.." position="after">
+                <field
+                    name="maintenance_plan_ids"
+                    nolabel="1"
+                    context="{'default_equipment_id': active_id, 'hide_equipment_id': 1}"
+                >
+                    <tree>
+                        <field name="maintenance_kind_id" string="Kind" />
+                        <field name="maintenance_team_id" string="Team" />
+                        <field name="name" />
+                        <field name="start_maintenance_date" string="Start Date" />
+                        <field name="interval" />
+                        <field name="interval_step" />
+                        <field name="duration" />
+                        <field name="next_maintenance_date" />
+                        <field
+                            name="maintenance_plan_horizon"
+                            string="P. Horizon period"
+                        />
+                        <field name="planning_step" string="P. Horizon step" />
+                    </tree>
+                </field>
+            </xpath>
```

Comment:
Not sure why the change from 

`<xpath expr="//group[@name='maintenance']/" position="after">`

`<xpath expr="//group[@name='maintenance']/.." position="after">`

at the very least I would add a <group string="Maintenance Plans">  around the whole thing, or alternatively remove the `nolabel` option